### PR TITLE
adding julia on slurm 

### PR DESCRIPTION
**Pull request by natalya-patrikeeva**
adding julia on slurm and deleting old julia job array page as it is absorbed into the combined slurm array page

### CODE REVIEW

File: docs/_user_guide/julia-slurm.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Running Julia on Yen-Slurm
```
Comment (mpj2104) [inline docs/_user_guide/julia-slurm.md:1]: When I edited the Slurm page, I changed all Yen-Slurm to Yen Slurm. Not sure which is better for consistency?

File: docs/_user_guide/julia-slurm.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Running Julia on Yen-Slurm 
+
+## Single-threaded Julia on Yen-Slurm
```
Comment (mpj2104) [inline docs/_user_guide/julia-slurm.md:3]: same question here with Yen-Slurm

File: docs/_user_guide/julia-slurm.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Similar to running Julia interactively, we will load Julia module but run the script using Slurm scheduler so that the script is executed on the Yen-Slurm nodes.
```
Comment (mpj2104) [inline docs/_user_guide/julia-slurm.md:4]: The only time I used the hyphen, was referring to `yen-slurm`. I don't feel particularly strongly about the notation, so maybe we choose one and stick with it.

File: docs/_user_guide/julia-slurm.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+# Run Julia script
+julia hello.jl
+```
+Then run it by submitting the job to the Slurm scheduler with:
+```title="Terminal Command"
+sbatch hello.slurm
+```
+
+## Multi-threaded Julia on Yen-Slurm
```
Comment (mpj2104) [inline docs/_user_guide/julia-slurm.md:30]: just flagging Yen-Slurm again

File: docs/_user_guide/julia-slurm.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+#SBATCH -t 1:00:00
+#SBATCH -o hello-%j.out
+#SBATCH --mail-type=ALL
+#SBATCH --mail-user=your_email@stanford.edu
+
+# Load software
+module load julia
+
+# Run Julia script
+julia hello.jl
```
Comment (mpj2104) [inline docs/_user_guide/julia-slurm.md:23]: Add an example `hello.jl` script somewhere?

**Review by mpj2104 — CHANGES_REQUESTED** [review]
Main question is about Yen-Slurm vs Yen Slurm

**Comment by natalya-patrikeeva** [comment]
i'll cancel this PR since we are combining all languages into one slurm page

**Pull request closed without merging**